pallets/metamask/README.md

```diff
@@ -0,0 +1,109 @@
+# Metamask pallet
```

Comment:
if I understand you correctly, `eth_signTypedData` is more general than metamask-specific api. should we consider name the pallet more generally as well?

Reply:
Yep, it is. But looks like only used by Metamask. At least I haven't seen any other uses of this API when looking for examples.
The purpose of this name was to combine some of the Metamask related features into single pallet.
Can be renamed. Names are welcomed.

Reply:
pallet-eip-712?